lib/rucio/web/rest/flaskapi/v1/main.py

```diff
@@ -44,6 +44,7 @@
     'rules',
     'scopes',
     'subscriptions',
+    'vos',
```

Comment:
Actually I don't know about this one. I think that should be removed again since it only makes sense in a MultiVO deployment

Reply:
@joeldierkes if you are still reading this, please take that one out :-)